tools/lambda-promtail/variables.tf

```diff
@@ -67,9 +67,9 @@ variable "extra_labels" {
 }
 
 variable "omit_extra_labels_prefix" {
-  type        = string
+  type        = bool
   description = "Whether or not to omit the prefix `__extra_` from extra labels defined in the variable `extra_labels`."
-  default     = "false"
+  default     = false
```

Comment:
Should we do the same change for the `keep_stream` variable?

Reply:
As I mentioned in the commit message: the only reason I changed this was that it hasn't been in a tagged release yet.

I avoided changing `keep_stream` as it has been around a while.